app/views/overview.html

```diff
@@ -67,7 +67,7 @@
     </div><!-- /.row -->
 
     <div class="row" ng-show="chart.data">
-        <div class="col-md-12">
+        <div ng-class="checks.length > 0 ? 'col-md-6 col-sm-6 col-xs-6' : 'col-md-12 col-sm-12 col-xs-12'">
```

Comment:
do it in Angular way: e.g. `ng-class="{strike: deleted, bold: important, 'has-error': error}"`